src/ray/gcs/gcs_client/test/service_based_gcs_client_test.cc

```diff
@@ -498,8 +488,8 @@ class ServiceBasedGcsClientTest : public ::testing::Test {
   // GCS server.
   gcs::GcsServerConfig config_;
   std::unique_ptr<gcs::GcsServer> gcs_server_;
-  std::unique_ptr<std::thread> server_io_service_thread_;
-  std::unique_ptr<boost::asio::io_service> server_io_service_;
+  size_t io_service_num_{2};
```

Comment:
Can we use a constant instead of 2? Maybe DEFAULT_GCS_NUM_IO_SERVICES with some comment? (Like we need a separate io service to avoid heartbeat delay)

Reply:
added in constants.h.